test/jdk/javax/naming/spi/providers/InitialContextTest.java

```diff
@@ -21,30 +21,48 @@
  * questions.
  */
 
-import javax.naming.Context;
-import java.io.*;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.io.SequenceInputStream;
+import java.io.StringWriter;
+import java.io.Writer;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.*;
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.Iterator;
 import java.util.List;
+import java.util.Map;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import static java.lang.String.format;
+import javax.naming.Context;
+
 import static java.util.Arrays.asList;
 import static java.util.Collections.singleton;
 import static java.util.Collections.singletonMap;
+import jdk.test.lib.process.OutputAnalyzer;
+import jdk.test.lib.process.ProcessTools;
 
 /*
  * @test
  * @bug 8044627
  * @summary Examines different ways JNDI providers can hook up themselves and
  *          become available. Each case mimics the most straightforward way of
  *          executing scenarios.
+ * @library /test/lib
+ * @build jdk.test.lib.process.ProcessTools
+ * @run main/othervm InitialContextTest
```

Comment:
Hello Mahendra, I see that this test and one other test is being changed to `/othervm` and I suspect it's because, we now call:

```java
System.setProperty("test.noclasspath", "true");
```
for the `ProcessTools` to skip adding the default `-cp` option to the launched Java process. 
In reality, we don't have to set that system property and thus you don't have to change this and the other test to `othervm`. See a previous discussion about the classpath handling by `ProcessTools` here https://github.com/openjdk/jdk/pull/17787/files/a9fcc6c2900356250b29b3d11b402790a84d9317#r1484276695 - essentially, whatever classpath you end up passing as part of the command to `ProcessTools.createTestJavaProcessBuilder()` will end up getting used.